Makefile.js

```diff
@@ -143,6 +143,20 @@ function generateBlogPost(releaseInfo) {
 }
 
 /**
+ * Generates a doc page with formatter result examples
+ * @param  {Object} formatterInfo Linting results from each formatter
+ * @returns {void}
+ */
+function generateFormatterExamples(formatterInfo) {
+    var output = ejs.render(cat("./templates/formatter-examples.md.ejs"), formatterInfo),
+        filename = "../eslint.github.io/docs/user-guide/formatters/index.md",
+        htmlFilename = "../eslint.github.io/docs/user-guide/formatters/html-formatter-example.html";
+
+    output.to(filename);
+    fs.writeFileSync(htmlFilename, formatterInfo.formatterResults.html.result);
```

Comment:
Yes, it's coming from ShellJS. And you're using it on the immediately preceding line (155). :) We should be consistent throughout.